### DISCUSSION TITLE
Terra (LUNA) + Terra Stablecoins

### DISCUSSION THREAD

**Discussion by trymeouteh** [MrNaif2018/bitcart#282] [Ideas]
Please consider supporting LUNA and Terra stablecoins.

Terra is a popular chain for stablecoins and unlike Tether, the is no trust in the backing of each coin.

EDIT: Terra is not a good option anymore due to the May Terra crash.

**Comment by MrNaif2018** [comment]
About lack of a decentralised stablecoin: on eth and bnb chains there is a DAI stablecoin which seems to be an algorithmical one, not backed up by tether's reserves. Also there is a separate support for xrg, a stablecoin based on bch which also is stable because of the algorithm

About Terra-heard about it once, will reply after I check more

**Comment by trymeouteh** [comment]
Terra is not a good option anymore due to the May Terra crash.

**Reply by MrNaif2018** [reply]
True (:
